documentation/developers/coreapps.md

```diff
@@ -21,7 +21,7 @@ For debugging, it is usually desirable to run the Jukebox directly from the cons
 
 ## Configuration Tools
 
-Before running the configuration tools, stop the Jukebox Core service.
+Before running the configuration tools, stop the Jukebox Core service `systemctl --user stop jukebox-daemon`. Make sure to start the service afterwards.
```

Comment:
Is this change neccessary? How to stop the service is already described in the link "best pratices". 
Shouldn't that (or another doc about the service where this is described too) be the place to link, instead of having this command cluttered throughout the docs?

Reply:
Yesterday when I followed the docs it seemed more convenient to me, but you are right it clutters the docs. 

I'll change it 🙂